src/store.ts

```diff
@@ -736,6 +741,13 @@ export function processFlags(config: SDKInitConfig): IFeatureFlags {
     flags[AudienceAPI] = config.flags[AudienceAPI] === 'True';
     flags[CaptureIntegrationSpecificIds] = config.flags[CaptureIntegrationSpecificIds] === 'True';
     flags[AstBackgroundEvents] = config.flags[AstBackgroundEvents] === 'True';
+    // default CollectClickIdV2Enabled to false when undefined
+    const collectV2 = config.flags[CollectClickIdV2Enabled] === 'True';
+    flags[CollectClickIdV2Enabled] = collectV2;
+    // Normalize mode to lower-case with default 'none'
+    const rawMode = (config.flags[IntegrationCaptureMode] || 'none').toString().trim().toLowerCase();
+    // Only meaningful when v2 is enabled; otherwise legacy boolean is used elsewhere
+    flags[IntegrationCaptureMode] = rawMode;
```

Comment:
Based on all the above comments and the fact that we are only using `captureIntegrationSpecificIds.V2`, I think we can get rid of a lot of this code and just have:

```suggestion
    flags['captureIntegrationSpecificIds.V2'] = config.flags['captureIntegrationSpecificIds.V2'] || null
```